apps/export/entries/excel_exporter.py

```diff
@@ -420,6 +420,7 @@ def add_entries(self, entries):
                 f'{lead.id}',
                 lead.title,
                 lead.url or (lead.attachment and lead.attachment.get_file_url()),
+                lead.get_authoring_organizations_type_display(),
```

Comment:
Also, add this `authors, authors parent, authors parent organization type, authors organization type` prefetch in `Entry.get_exportable_queryset)` as well.